detect_utm_zone.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+
+"""
+Accepts a polygon shapefile in WGS84 and finds the SRID for the UTM zone
+Based on https://github.com/jbranigan/geo-scripts-python/blob/master/latlng2utm/detect-utm-zone.py
+"""
+
+from osgeo import ogr
+import math
+import argparse
+
+
+def check_latlng(check_bbox):
+    ''' Checks to see if the file coordinates are in lat/lng '''
```

Comment:
Switch to triple double-quotes for consistency with PEP257, both here and throughout the file